### ISSUE TITLE
requires.jl static arrays warning

### ISSUE THREAD

**Issue by anandijain** [JuliaArrays/ArrayInterface.jl#148]
```julia
┌ Warning: Error requiring `StaticArrays` from `ArrayInterface`
--
  | │   exception =
  | │    too many parameters for type
  | │    Stacktrace:
  | │      [1] top-level scope
  | │        @ ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/ArrayInterface/GV582/src/ArrayInterface.jl:930
  | │      [2] eval
  | │        @ ./boot.jl:360 [inlined]
  | │      [3] eval
  | │        @ ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/ArrayInterface/GV582/src/ArrayInterface.jl:1 [inlined]
  | │      [4] (::ArrayInterface.var"#53#80")()
  | │        @ ArrayInterface ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:99
  | │      [5] err(f::Any, listener::Module, modname::String)
  | │        @ Requires ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:47
  | │      [6] (::ArrayInterface.var"#52#79")()
  | │        @ ArrayInterface ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:98
  | │      [7] withpath(f::Any, path::String)
  | │        @ Requires ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:37
  | │      [8] (::ArrayInterface.var"#51#78")()
  | │        @ ArrayInterface ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:97
  | │      [9] listenpkg(f::Any, pkg::Base.PkgId)
  | │        @ Requires ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:20
  | │     [10] macro expansion
  | │        @ ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:95 [inlined]
  | │     [11] __init__()
  | │        @ ArrayInterface ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/ArrayInterface/GV582/src/ArrayInterface.jl:869
  | │     [12] _include_from_serialized(path::String, depmods::Vector{Any})
  | │        @ Base ./loading.jl:674
  | │     [13] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
  | │        @ Base ./loading.jl:760
  | │     [14] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
  | │        @ Base ./loading.jl:689
  | │     [15] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
  | │        @ Base ./loading.jl:749
  | │     [16] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
  | │        @ Base ./loading.jl:689
  | │     [17] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
  | │        @ Base ./loading.jl:749
  | │     [18] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
  | │        @ Base ./loading.jl:689
  | │     [19] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
  | │        @ Base ./loading.jl:749
  | │     [20] _tryrequire_from_serialized(modkey::Base.PkgId, build_id::UInt64, modpath::String)
  | │        @ Base ./loading.jl:689
  | │     [21] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String)
  | │        @ Base ./loading.jl:749
  | │     [22] _require(pkg::Base.PkgId)
  | │        @ Base ./loading.jl:998
  | │     [23] require(uuidkey::Base.PkgId)
  | │        @ Base ./loading.jl:914
  | │     [24] require(into::Module, mod::Symbol)
  | │        @ Base ./loading.jl:901
  | │     [25] include(mod::Module, _path::String)
  | │        @ Base ./Base.jl:386
  | │     [26] exec_options(opts::Base.JLOptions)
  | │        @ Base ./client.jl:285
  | │     [27] _start()
  | │        @ Base ./client.jl:485
  | └ @ Requires ~/.cache/julia-buildkite-plugin/depots/609dd3f6-6d58-4fb2-a8ce-f94307987da5/packages/Requires/7Ncym/src/require.jl:49
```
Chris R told me to ping you @chriselrod

**Comment by chriselrod** [comment]
Which version of StaticArrays and which version of ArrayInterface?
That should be fixed with ArrayInterface 3.1.9 (the latest release).

**Comment by anandijain** [comment]
Another classic case of forgetting to `]up`. Sorry and thanks
